tattoo.ts

```diff
@@ -243,12 +277,25 @@ function standardProgressBar(label: string, total: number) {
  *
  * returns a promise of the nodegit Branch object for the new branch.
  */
-async function checkoutNewBranch(
-    repo: nodegit.Repository, branchName: string): Promise<void> {
-  const commit = await repo.getHeadCommit();
-  const branch =
-      await nodegit.Branch.create(repo, branchName, commit, false);
-  return repo.checkoutBranch(branch);
+async function checkoutBranch(
+    repo: Promise<nodegit.Repository>, branchName: string): Promise<nodegit.Repository> {
+  let r = await repo;
+  try {
+  await new Promise((resolve, reject) => (
+    child_process.exec("git checkout " + branchName,
+        {cwd: r.workdir()},
+        (error, stdout, stderr)  => {
+          if (error) {
+            reject(error);
+          } else {
+            resolve();
+          }
+        })
+    ));
+  } catch (err) {
+    console.log("Error checkout out " + branchName + "in : " + r.workdir());
```

Comment:
I'm not sure this error should be caught. Can we really proceed if a checkout fails?